src/Microsoft.AspNetCore.Mvc.TagHelpers/InputTagHelper.cs

```diff
@@ -93,9 +93,8 @@ public override int Order
         public ModelExpression For { get; set; }
 
         /// <summary>
-        /// The composite format <see cref="string"/> (see http://msdn.microsoft.com/en-us/library/txafckwd.aspx) to
-        /// apply when converting the <see cref="For"/> result to a <see cref="string"/>. Sets the generated "value"
-        /// attribute to that formatted <see cref="string"/>.
+        /// The format string (see https://msdn.microsoft.com/en-us/library/txafckwd.aspx) used to format the
+        /// <see cref="For"/> result. Sets the generated "value" attribute to that formatted string.
```

Comment:
This rewording was intentional but I'll revert if @Eilon thinks it's a regression in readability.